ansible/roles/sidekiq/tasks/main.yml

```diff
@@ -0,0 +1,44 @@
+---
+  - name: Install monit
+    apt:
+      name: monit
+      state: present
+    sudo: true
+
+  - name: Install sidekiq dependencies
+    apt:
+      name: "{{item}}"
+      state: present
+    with_items:
+      - redis-server
+    sudo: true
+
+  - name: Install sidekiq monit script
+    template:
+      src: sidekiq-monit-rc
+      dest: /etc/monit/conf.d/sidekiq_{{project_name}}_{{rails_env}}
+    sudo: true
```

Comment:
I wanted to start doing a thing where the roles would declare a new "fact" like "monit_installed" and then you could put an if: monit_installed on this task